opentech/apply/funds/views.py

```diff
@@ -316,7 +342,7 @@ class SubmissionDetailView(ViewDispatcher):
     applicant_view = ApplicantSubmissionDetailView
 
     def admin_check(self, request):
-        if request.user.is_reviewer:
+        if request.user.is_reviewer or request.user.is_partner:
```

Comment:
@Parbhat I need to check not only that the user is a partner but also that they are assigned as a partner on this specific submission.

How do you recommend I do that best? The submission object does not seem to be available at this stage.

Reply:
I don't think you want to give partners the admin view of the application. They should have the same access rights as a normal applicant?

Reply:
OTF has requested a mayor change to what reviewers can see but I think that is outside of this ticket.

This access solution is needed either way so solve it in this pr and then we take on separating the views etc. in another pr.